pigpen-cascading/src/main/clojure/pigpen/cascading/core.clj

```diff
@@ -1,16 +0,0 @@
-(ns pigpen.cascading.core
```

Comment:
Why did you get rid of this one? I like having this so that we can expose only the fns we want to make part of the public API